pkgs/by-name/ag/age-plugin-se/package.nix

```
@@ -0,0 +1,63 @@
+{
+  lib,
+  fetchFromGitHub,
+  swiftPackages,
+  swift,
+  swiftpm,
+  nix-update-script,
+}:
+let
+  stdenv = swiftPackages.stdenv;
+in
+stdenv.mkDerivation (finalAttrs: {
+  pname = "age-plugin-se";
+  version = "0.1.4";
+
+  src = fetchFromGitHub {
+    owner = "remko";
+    repo = "age-plugin-se";
+    tag = "v${finalAttrs.version}";
+    hash = "sha256-sg73DzlW4aXNbIIePZox4JkF10OfsMtPw0q/0DWwgDk=";
+  };
+
+  nativeBuildInputs = [
+    swift
+    swiftpm
+  ];
+
+  patchPhase =
+    let
+      swift-crypto = fetchFromGitHub {
+        owner = "apple";
+        repo = "swift-crypto";
+        # FIXME: Update to a newer version once https://github.com/NixOS/nixpkgs/issues/343210 is fixed
+        # This is the last version to support swift tools 5.8 which is newest version supported by nixpkgs:
+        # https://github.com/apple/swift-crypto/commit/35703579f63c2518fc929a1ce49805ba6134137c
+        rev = "3.7.1";
```

```suggestion
        tag = "3.7.1";
```

sorry, I missed this one as well. And you can squash it all into one commit IMO, this is a trivial change that I don't need credit for

---

Done. Thanks for the review and teaching about the `tag` keyword here 💯